jsapp/js/account/usage/usageProjectBreakdown.module.scss

```diff
@@ -10,6 +10,27 @@
   overflow-x: auto;
 }
 
+.intervalBanner {
+  background-color: colors.$kobo-light-blue;
```

Comment:
I think this should be `colors.$kobo-bg-blue`. See [figma](https://www.figma.com/design/NonWWcfadc3QfS3q3D8hRB/⚙%EF%B8%8F-KPI-%2F-Account-Settings?node-id=5132-14047&t=t4lq34Sd8JYgAGd2-0)